client/lib/domains/resolve-domain-status.tsx

```diff
@@ -135,27 +135,14 @@ export function resolveDomainStatus(
 						: translate( 'Expiring soon' );
 				}
 
-				if ( isExpiringSoon( domain, 7 ) ) {
-					return {
-						statusText: expiresMessage,
-						statusClass: `status-${ domain.autoRenewing ? 'success' : 'error' }`,
-						status: status,
-						icon: 'info',
-						listStatusText: expiresMessage,
-						listStatusClass: domain.autoRenewing ? 'info' : 'alert',
-						listStatusWeight: 1000,
-						noticeText,
-					};
-				}
-
 				return {
 					statusText: expiresMessage,
-					statusClass: 'status-warning',
+					statusClass: `status-${ domain.autoRenewing ? 'success' : 'error' }`,
```

Comment:
The CSS classes `domain-row__status-error-dot` and `domain-row__status-warning-dot` use the same dot color, so this change is ok